examples/layer-browser/src/app.js

```diff
@@ -111,6 +111,12 @@ class App extends PureComponent {
     this.setState({clickedItem: info});
   }
 
+  _onQueryObjects() {
+    const {width, height} = this.state;
+    const infos = this.refs.deckgl.queryObjects({x: 0, y: 0, width, height});
```

Comment:
I know this is just an example, but still, can we change the query boundary to be something like, {x: width / 4, y: height / 4, width / 4 * 3, height / 4 * 3} so that this will return only part of the objects current on screen? It might serve partially as a sanity test